### PR TITLE
Document creating an API key from the CLI

### PR DESCRIPTION
**Pull request by lonix1**
Related to https://github.com/go-gitea/gitea/issues/25503

### CODE REVIEW

File: docs/content/doc/administration/command-line.en-us.md [hunk 1]
```diff
@@ -108,6 +108,14 @@ Admin operations:
         - `--all`, `-A`: Force a password change for all users
         - `--exclude username`, `-e username`: Exclude the given user. Can be set multiple times.
         - `--unset`: Revoke forced password change for the given users
+    - `generate-access-token`:
+      - Options:
+        - `--username value`, `-u value`: Username. Required.
+        - `--token-name value`, `-t value`: Token name. Required.
+        - `--scopes value`: Comma-separated list of scopes.
```
Comment (delvh) [inline docs/content/doc/administration/command-line.en-us.md]:
```suggestion
        - `--scopes value`: Comma-separated list of scopes. Scopes follow the format `[read|write]:<block>` or `all` where `<block>` is one of the available visual groups you can see when opening the API page showing the available routes (for example `repo`).
```

**Review by delvh — APPROVED** [review]

**Comment by delvh** [comment]
Oh, by the way, please cherrypick this PR also to `main` once it is merged.
Otherwise, it would be lost once 1.20 is outdated.

**Comment by delvh** [comment]
> I hope I clicked the right button!

Yup, that was the correct one.

**Comment by KN4CK3R** [comment]
You targeted the 1.20 branch instead of `main`. Usually changed are made in the `main` branch and backported into the other relevant branches and not the other way around.

**Review by lunny — APPROVED** [review]

**Comment by techknowlogick** [comment]
@lonix1 I will cherry pick it for you. Thanks for your contribution :)

**Pull request merged**